Add announcement for SQL training series (#535)
trinodb/trino.io: _posts/2023-09-08-training-series.md

---
layout: post
title: "Learning SQL with Trino from the experts"
author: "Manfred Moser"
excerpt_separator: <!--more-->
image: 
show_pagenav: false
---

> Do you think SQL stands for Silly Quacking Leopards and write all your code in

Reviewer: It's generally poor practice to start an article with something that a reader can respond "no" to. I know it's a joke (and a good one), but if I have even a little bit of SQL knowledge, I'm immediately going to assume that this content is below my level. I might move it a couple lines deeper in so it's not immediately going to steer readers away.

Author: Good point .. let me rejig .. also .. I can link to the book .. since the cover is a leopardfrog (need to confirm that).

Java, Python, or Go? But you need to get data out of object storage in the cloud
and some relational database at the same time? You should look at Trino and
learning about SQL.

Or do you know that ins and outs of joins, window functions, and your SQL

> queries are counted by the pages and not lines? You are the expert on SQL in
> your database team. You should also look at Trino and SQL.

Reviewer: ```suggestion
queries are counted by the pages and not lines? You may even be the expert on SQL
on your team. You should *also* look at Trino and SQL.
```

Author: Done

> Luckily for you all we got the right SQL training coming up for everyone in our

Reviewer: ```suggestion
Luckily for you all, we have the right SQL training coming up for everyone in our
```

> upcoming series with the founders of the Trino project and SQL experts Martin

Reviewer: Should say "coming up" or "upcoming," but not both.

Author: Done

Traverso, Dain Sundstrom, and David Phillips, and myself as host and co-trainer.

<!--more-->

> In the SQL training series we start with the basics around Trino, so you we are

Reviewer: ```suggestion
In the SQL training series, we start with the basics of Trino, so that we're
```

Reviewer: I'd also consider saying "we'll start," as the future tense is fine here. We're making a promise, and it sounds more natural that way.

Author: Reworded all that some more.

all on the same page. You will know the benefits of connecting [many data
sources]({{site_url}}/ecosystem.html#data-sources) to Trino, and using
[different clients]({{site_url}}/ecosystem.html#clients), but always using the
same powerful SQL. And for the SQL pros, you learn about catalogs and queries
that go across data sources.

> Then we glance at the really basic SQL foundations, since there are literally

Reviewer: ```suggestion
Then we'll glance at the basic SQL foundations, since there are literally
```
Possibly

Author: ok

hundreds of books, videos, and training course around. All of them teach you
things like `SELECT` statements, and `WHERE` clauses, and unravel the confusions
around `LEFT OUTER JOIN` and the like.

> And thats where we are getting to the interesting stuff. Following is a list of

Reviewer: ```suggestion
And after that is when we get to the interesting stuff. Following is a list of
```

> some of stuff we will cover:

Reviewer: ```suggestion
some of the topics we will cover:
```
And we _are_ using the future tense here, so we should make it consistent one way or the other.

Author: Done

---
layout: post
title: "Learning SQL with Trino from the experts"
author: "Manfred Moser"
excerpt_separator: <!--more-->
image: /assets/trino-sql.png
show_pagenav: false
---

> Do you have a rough idea what SQL? Do you need to get data out of object storage

Reviewer: ```suggestion
Do you have a rough idea of what SQL is? Do you need to get data out of object storage
```

Author: Applied

in the cloud and some relational database at the same time? You should look at
Trino and learning about SQL.

> Or do you know that ins and outs of joins, window functions, and your SQL

Reviewer: ```suggestion
Or do you know the ins and outs of joins, window functions, and your SQL
```

Author: Applied.
